src/quest_manager/models.py

```diff
@@ -439,13 +456,40 @@ def get_available_without_course(self, user):
         return qs.not_in_progress_completed_or_cooldown(user)
 
     def all_drafts(self, user):
-        qs = self.get_queryset().filter(visible_to_students=False)
```

Comment:
I don't understand why you changed the default in get_queryset, and then proceeded to pass the opposite parameter in all these methods!